__tests__/server.test.js

```diff
@@ -144,4 +144,56 @@ describe("Testing APIs for GET /api/articles/:article_id", () => {
   
   
       })
-    })
\ No newline at end of file
+    })
+
+    describe("Testing APIs for GET /api/articles", () => {
+      test("Status 200 and return an array of comments for the specified article_id", () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) =>{
+  
+     
+          const arrayOfComments = body
+  
+        
+  
+          expect(arrayOfComments.length).toBe(11)
+  
+          arrayOfComments.forEach((comment)=>{
+            expect(comment).toMatchObject({
+              author:expect.any(String),
+              comment_id:expect.any(Number),
+              body:expect.any(String),
+              article_id:expect.any(Number),
+              created_at:expect.any(String),
+              votes:expect.any(Number),
+            })
+          })
```

Comment:
what about testing for 200 for an article with no comments?  this will require you to check if an article exists first. I think this was the topic of today's lecture.